Core/tests/Unit/Upload/ResumableUploaderTest.php

```diff
@@ -173,6 +173,33 @@ public function testResumeFinishedUpload()
         );
     }
 
+    public function testRetryOptionsPassing()
+    {
+        // This tests whether retry related options are properly set in the
```

Comment:
I like the description.
nit: Put this as a docbloc above the method.
Suggested: Do the same for the other tests in this PR.

Reply:
Sound good.
Updated docbloc. [9e155ff](https://github.com/googleapis/google-cloud-php/pull/5953/commits/9e155ff90545e3b7a65efa01492261db4cdc874e)